CONTRIB.rst

```diff
@@ -234,6 +234,7 @@ please open an issue on GitHub or mention it on the mailing list.
 - Peter Kerpedjiev <https://github.com/pkerpedjiev>
 - Peter Slickers <piet at domain clondiag.com>
 - Philip Bergstrom <https://github.com/phber>
+- Philip Capel <insert.available.name at gmail>
```

Comment:
You'd rather this than <https://github.com/pcapel>?

Reply:
This would be my first time adding myself to a file like this, so I wasn't entirely sure which would be best. I suppose pointing back to my github profile is more practical.